reporters/GlobalSummaryReporter.js

```diff
@@ -0,0 +1,249 @@
+      if (event.blockNumber >= startBlockNumberForPeriod) {
+        collateralWithdrawnPeriod = collateralWithdrawnPeriod.add(toBN(event.amount));
+      }
+    }
+
+    allSponsorStatsTable["collateral withdrawn"] = {
+      cumulative: this.formatDecimalString(collateralWithdrawn),
+      [periodLabelInHours]: this.formatDecimalString(collateralWithdrawnPeriod)
+    };
+
+    // - Net collateral deposited into contract:
+    let netCollateralWithdrawn = collateralDeposited.sub(collateralWithdrawn);
+    let netCollateralWithdrawnPeriod = collateralDepositedPeriod.sub(collateralWithdrawnPeriod);
+    allSponsorStatsTable["net collateral deposited"] = {
+      cumulative: this.formatDecimalString(netCollateralWithdrawn),
+      [periodLabelInHours]: this.formatDecimalString(netCollateralWithdrawnPeriod)
+    };
+
+    // - Tokens minted: Creates
+    let tokensMinted = toBN("0");
+    let tokensMintedPeriod = toBN("0");
+    for (let event of createEvents) {
+      tokensMinted = tokensMinted.add(toBN(event.tokenAmount));
+      if (event.blockNumber >= startBlockNumberForPeriod) {
+        tokensMintedPeriod = tokensMintedPeriod.add(toBN(event.tokenAmount));
+      }
+    }
+    allSponsorStatsTable["tokens minted"] = {
+      cumulative: this.formatDecimalString(tokensMinted),
+      [periodLabelInHours]: this.formatDecimalString(tokensMintedPeriod),
+      current: this.formatDecimalString(await this.empContract.methods.totalTokensOutstanding().call())
+    };
+
+    // - Tokens burned: Redeems, SettleExpired's, Liquidations
+    let tokensBurned = toBN("0");
+    let tokensBurnedPeriod = toBN("0");
+    for (let event of redeemEvents) {
+      tokensBurned = tokensBurned.add(toBN(event.tokenAmount));
+      if (event.blockNumber >= startBlockNumberForPeriod) {
+        tokensBurnedPeriod = tokensBurnedPeriod.add(toBN(event.tokenAmount));
+      }
+    }
+    for (let event of expirySettlementEvents) {
+      tokensBurned = tokensBurned.add(toBN(event.tokensBurned));
+      if (event.blockNumber >= startBlockNumberForPeriod) {
+        tokensBurnedPeriod = tokensBurnedPeriod.add(toBN(event.tokensBurned));
+      }
+    }
+    for (let event of liquidationEvents) {
+      tokensBurned = tokensBurned.add(toBN(event.tokensOutstanding));
+      if (event.blockNumber >= startBlockNumberForPeriod) {
+        tokensBurnedPeriod = tokensBurnedPeriod.add(toBN(event.tokensOutstanding));
+      }
+    }
+    allSponsorStatsTable["tokens burned"] = {
+      cumulative: this.formatDecimalString(tokensBurned),
+      [periodLabelInHours]: this.formatDecimalString(tokensBurnedPeriod)
+    };
+
+    // - Net tokens minted:
+    let netTokensMinted = tokensMinted.sub(tokensBurned);
+    let netTokensMintedPeriod = tokensMintedPeriod.sub(tokensBurnedPeriod);
+    allSponsorStatsTable["net tokens minted"] = {
+      cumulative: this.formatDecimalString(netTokensMinted),
+      [periodLabelInHours]: this.formatDecimalString(netTokensMintedPeriod)
+    };
```

Comment:
Similar to my comment above -- a potentially simpler way to track token burns is to just use the `Transfer` event on the synthetic token and look for events that are `from: emp.address` and `to: 0x0`.

If you think that tracking the events you've used will be useful elsewhere, then I'm totally cool keeping this as is.

Reply:
Same comment as above